### DISCUSSION TITLE
Open on Startup

### DISCUSSION THREAD

**Discussion by mahmoudtark25** [cstayyab/WALC#78] [Feature Requests]
setting option for starting hidden at system startup.

**Comment by daffamumtaz2361** [comment]
Can't you just add it on your Startup Applications or Autostart on your DE? That's what I do in Plasma

**Comment by mahmoudtark25** [comment]
> Can't you just add it on your Startup Applications or Autostart on your DE? That's what I do in Plasma

I have already done that but it will be easier and better for many users if it is included in app settings.

**Reply by madroots** [reply]
Yes and make it start with the system by default. right? Go install windows. Everything will then autostart when you boot

**Comment by amanharwara** [comment]
[Teamwork/node-auto-launch](https://github.com/Teamwork/node-auto-launch) might be useful for this.

**Comment by douglasodev** [comment]
> setting option for starting hidden at system startup.

Do you talk about the arguments set after the bin app like this bellow ?

env BAMF_DESKTOP_FILE_HINT=/var/lib/snapd/desktop/applications/telegram-desktop_telegramdesktop.desktop /snap/bin/telegram-desktop **-startintray -- %u**

This would be a great feature that WALC needs to start minimized. Please devs, set flags/arguments on the binaries to WALC. Thank you!

**Comment by daffamumtaz2361** [comment]
But there's already an option to start on tray. Why would WALC need an argument?

Personally this feature are low priority to me. I probably not going to implement it anytime soon. But you are welcome to open a pr
